cmd/gaia/cli_test/test_helpers.go

```diff
@@ -322,6 +322,12 @@ func (f *Fixtures) TxEncode(fileName string, flags ...string) (bool, string, str
 	return executeWriteRetStdStreams(f.T, addFlags(cmd, flags), app.DefaultKeyPass)
 }
 
+// TxDecode is gaiacli tx decode
+func (f *Fixtures) TxDecode(tx string, flags ...string) (bool, string, string) {
+	cmd := fmt.Sprintf("../../../build/gaiacli tx decode %s %v", tx, f.Flags())
```

Comment:
We don't do `../../../build/gaiacli` anymore. Please either rebase or merge develop in.

Reply:
Done!
- app.DefaultKeyPass -> client.DefaultKeyPass
- ../../../build/gaiacli ->  f.GaiacliBinary